update inkit docs per partner request (#4663)
segmentio/segment-docs: src/connections/destinations/catalog/inkit/index.md

rewrite: true
id: 5f0746ced1c79b49ddee49fd
---

> [Inkit](https://inkit.com) and Segment empower organizations to securely generate and distribute documents - both digitally as well as via direct mail. 

Reviewer: ```suggestion
[Inkit](https://inkit.com){:target="_blank"} empowers organizations to securely generate and distribute documents - both digitally as well as through direct mail. 
```

> For example, automatically create and send electronic documents like invoices, reports, notices, and more via a magic link or e-delivery. Or generate and send documents for e-signature, storage, postcards, letters, and more, all powered by the Inkit integration for Segment.

Reviewer: ```suggestion
For example, automatically create and send electronic documents like invoices, reports, notices, and more using a magic link or e-delivery. Or generate and send documents for e-signature, storage, postcards, letters, and more, all powered by the Inkit integration for Segment.
```

> note ""
> Inkit maintains this destination. For any issues with the destination, [email the Inkit support team](mailto:support@inkit.com).

## Getting Started

Add the destination:

1. From the Destinations catalog page in your Segment workspace, click **Add Destination**.
2. Search for "Inkit" in the Destinations Catalog, and select the "Inkit" destination.
3. Choose which Source should send data to the "Inkit" destination.

Get the Inkit API Key:

> 1. [Sign up](https://app.inkit.com/auth-init) and create an Inkit account.

Reviewer: ```suggestion
1. [Sign up](https://app.inkit.com/auth-init){:target="_blank"} and create an Inkit account.
```

> 2. Follow the instructions in the documentation to [create an API key](https://docs.inkit.com/docs/add-an-api-key-to-your-account).

Reviewer: ```suggestion
2. Follow [the instructions create an API key](https://docs.inkit.com/docs/add-an-api-key-to-your-account){:target="_blank"}.
```

3.	Enter the “API Key” in the “Inkit” destination settings in Segment.

To use a Template ID:

1.	From the Destinations catalog page in your Segment workspace, click Add Destination.
2.	Search for “Inkit” in the Destinations Catalog, and select the “Inkit” destination.
3.	Choose which Source should send data to the “Inkit” destination.
4.	[Create a template](https://docs.inkit.com/docs/create-a-template) in Word, PDF, HTML, Excel, or PowerPoint.
5.	Copy the Template ID to the “Templates” tab in the Inkit web app.
6.	Paste the id into the “template_id” field when setting up the Destination in Segment.

> For more information, see Inkit [documentation](https://docs.inkit.com/docs/welcome-to-inkit).

Reviewer: ```suggestion
For more information, see the Inkit [documentation](https://docs.inkit.com/docs/welcome-to-inkit){:target="_blank"}.
```
